io_ogre/util.py

```diff
@@ -9,11 +9,25 @@
 import logging
 import subprocess
 import re
+import sys
 
 def xml_converter_parameters():
     """
     Return the name of the ogre converter
     """
+    if sys.platform.startswith("win"):
+        # Don't display the Windows GPF dialog if the invoked program dies.
+        # See comp.os.ms-windows.programmer.win32
+        # How to suppress crash notification dialog?, Jan 14,2004 -
+        # Raymond Chen's response [1]
+
+        import ctypes
+        SEM_NOGPFAULTERRORBOX = 0x0002 # From MSDN
+        ctypes.windll.kernel32.SetErrorMode(SEM_NOGPFAULTERRORBOX);
+        subprocess_flags = 0x8000000 #win32con.CREATE_NO_WINDOW?
```

Comment:
subprocess_flags is unused. if it works for you like this, you can probably remove it.

Reply:
I tried it out.  Works fine without subprocess_flags.  I removed those lines.